app/helpers/teacher_sets_helper.rb

```diff
@@ -1,4 +1,61 @@
 # frozen_string_literal: true
 
 module TeacherSetsHelper
+  def teacher_set_info(ts_obj)
```

Comment:
I'd like to see some method documentation.  
- what does this method do
- what parameters does it expect
- what is the expected output
- what is the output on any errors